pkg/controller/ingress/ingress_controller_test.go

```diff
@@ -242,6 +272,39 @@ func TestRouteMigration(t *testing.T) {
 
 		routes := routeList.Items
 		assert.ElementsMatch(t, routes, test.want)
+
+		// A ServiceMeshMemberRole resource with metadata
+		smmrDelete := &maistrav1.ServiceMeshMemberRoll{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      smmrName,
+				Namespace: serviceMeshNamespace,
+			},
```

Comment:
This test is broken. You start with an empty SMMR to begin with so the test below will always pass. This should contain the namespace of `deleteIngress` and then be empty after the reconcile loop.